ckanext/canada/tables/briefingt.yaml

```diff
@@ -187,6 +198,7 @@ resources:
       en: This field will display any additional information/comments for the Briefing Note, in English.
```

Comment:
i think it should be 'French'?

Reply:
Do you know where i can get a translation?